src/app/runtime/runtime.wac.ts

```diff
@@ -101,18 +92,29 @@ export class WACRuntime implements Runtime {
         return this._connecting
     }
 
+    public PrepareIISHost(p: any): Observable<any> {
+        return this.powershellService.run(PowerShellScripts.admin_api_util, p).map((status: HostStatus) => {

```

Comment:
Can we add some timeout here such as 60 seconds?

Reply:
Not exactly sure what timeout to set here. Installation can take arbitrary log depends on the network though. Instead of defining a timeout now maybe we should look at user feedback